core/schema/modulesource.go

```diff
@@ -1808,6 +1810,8 @@ func (s *moduleSourceSchema) runCodegen(
 	return genDirInst, nil
 }
 
+var ErrSDKClientGeneratorNotImplemented = errors.New("the SDK does not implement the `ClientGenerator` interface")
```

Comment:
nit: group all the error types at the top of the file.

Reply:
Done